src/App.js

```diff
@@ -1,27 +1,24 @@
-import logo from './logo.svg';
 import './App.css';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import  Logo from './assets/logo.svg';
+import Header from './components/headerComponent';
+import Footer from './components/footerComponent';
+import Slider from './components/slider/sliderComponent';
+import FeaturedBanners from './mocks/en-us/featured-banners.json';
+import FeaturedProducts from './components/featuredProductsComponent';
+import Categories from './mocks/en-us/product-categories.json';
+import Products from './mocks/en-us/featured-products.json';
```

Comment:
I think is best to change some of this imports name to something that allows you to know they are mocks
```suggestion
import featuredBannersMock from './mocks/en-us/featured-banners.json';
import featuredProductsMock from './components/featuredProductsComponent';
import Categories from './mocks/en-us/product-categories.json';
import productsMock from './mocks/en-us/featured-products.json';
```